### PR TITLE
You tube meta

### PR DESCRIPTION
**Pull request by mitenevav**

### CODE REVIEW

File: Pipeline/Preprocessing/YouTubeMetaExtraction.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        return songs
+
+    def get_caption(self, language_code):
+        """
+        Getter for the caption to the video
+        :param language_code: language code
+        :return: caption (str)
+        """
+        cap = self.__video.captions.get_by_language_code(language_code)
+        assert cap is not None, "Invalid language code"
```
Comment (OlegBEZb) [inline Pipeline/Preprocessing/YouTubeMetaExtraction.py:69]: Are other errors possible?
From my point of view, if you make a specific assertion about an argument, you need to check it directly

File: Pipeline/Preprocessing/YouTubeMetaExtraction.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        Getter for the parts from caption with a music tag
+        :param language_code: language code
+        :return: times of parts (list)
+        """
+        lang = {'en': 'music', 'ru': 'музыка'}
+        if language_code not in lang:
+            return None
+
+        cap = self.__video.captions.get_by_language_code(language_code)
+        assert cap is not None, "Invalid language code"
```
Comment (OlegBEZb) [inline Pipeline/Preprocessing/YouTubeMetaExtraction.py:90]: please, you previously defined function

Comment (mitenevav) [reply Pipeline/Preprocessing/YouTubeMetaExtraction.py]: replaced it with a function call

File: Pipeline/Preprocessing/YouTubeMetaExtraction.py (hunk 1 above)
Comment (mitenevav) [reply Pipeline/Preprocessing/YouTubeMetaExtraction.py:76]: added more checks

**Pull request merged**